packages/main/src/SelectPopover.hbs

```diff
@@ -1,10 +1,12 @@
 {{#if options}}
 	<ui5-responsive-popover
+		{{!-- tabindex -1 is needed to keep the popover out of tab chain --}}
+		tabindex="-1"
 		hide-arrow
 		_disable-initial-focus
 		placement="Bottom"
 		class="ui5-select-popover {{classes.popover}}"
-		horizontal-align="Start"
+		horizontal-align="Left"
```

Comment:
ui5-popover => Use `Start` instead of `Left`
ui5-list => Use `SingleAuto` instead of `SingleSelectAuto`